test/test-files/step-function.ts

```diff
@@ -6,5 +6,30 @@ const app = new App({
 });
 const stack = new Stack(app, "stack");
 
-// arithmetic is not supported by Amazon States Language
-new StepFunction(stack, "F", () => 1 + 2);
+// unsupported arithmetic
```

Comment:
nit: plus objects. arrays, symbols, and functions?

Reply:
Anything constantly known should be supported.